esmvaltool/recipes/recipe_seaice.yml

```diff
@@ -0,0 +1,151 @@
+# ESMValTool
+# recipe_seaice.yml
+---
+documentation:
+  description: |
+    Recipe for plotting sea ice diagnostics at the Arctic and Antarctic.
+
+  authors:
+    - senf_da
+
+  maintainer:
+    - laue_ax
+
+  references:
+    - stroeve07grl
+
+  projects:
+    - esmval
+
+preprocessors:
+  pp_noop:
```

Comment:
There is no need to define a preprocessor if you just use the default preprocessing functions. This can be removed.